assets/src/js/cookies.js

```diff
@@ -1,8 +1,8 @@
 /* global dataLayer */
 export const setupCookies = () => {
   window.dataLayer = window.dataLayer || [];
-  const ENABLE_ANALYTICAL_COOKIES = window.p4bk_vars.enable_analytical_cookies;
-  const ENABLE_GOOGLE_CONSENT_MODE = window.p4bk_vars.enable_google_consent_mode;
+  const ENABLE_ANALYTICAL_COOKIES = window.p4_vars.enable_analytical_cookies || false;
```

Comment:
What happens if the `Activate all blocks` feature is disabled? I think we don't set these variables then 🤔

Reply:
Gotcha! 
But my point is whenever we finish the moving from plugin to the theme, I assume we're not going to use `p4bk_vars` anymore. Am I missing something else?

Reply:
Indeed we should not use them anymore then

Reply:
But we'll need to replace and use `p4_vars` instead, am I correct?